src/components/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,49 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # AVATAR
+\*------------------------------------*/
+
+/**
+ * Avatar
+ */
+.avatar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  overflow: hidden;
+
+  color: var(--eds-theme-color-text-neutral-strong);
+  background-color: var(--eds-theme-color-background-neutral-medium);
+}
+
+.avatar:focus {
+  outline: var(--eds-border-width-sm) solid var(--eds-theme-color-focus-ring);
+  outline-offset: 2px;
+}
+
+.avatar--sm {
+  @mixin eds-theme-typography-label-md;
+
+  height: 32px;
```

Comment:
nit: should these be size tokens or just rems?

Reply:
Correct, i will test a few things with the sizing. We may have the proper values in tokens already, and their usages here are tier 3

Reply:
@jinlee93 the sizes here correspond to `eds-size-4` thru `eds-size-6` for each size